app/controllers/contacts_controller.rb

```diff
@@ -50,8 +50,9 @@ class ContactsController < ApplicationController
       ContactWrt.new(
         name: requestor_details[:name],
         your_email: requestor_details[:email],
+        wca_id: User.find_by(email: requestor_details[:email])&.wca_id || 'None',
```

Comment:
Why don't you let the user enter the WCA ID in the frontend for themselves?

Reply:
If user is already logged-in, we will already have that data. We can get WCA ID from user if the user is not logged in. I'll consider that in another PR later, hope that is fine.